article.go

```diff
@@ -19,7 +19,7 @@ import (
 	blackfriday "gopkg.in/russross/blackfriday.v2"
 )
 
-// Author is a struct with contributor info
+// Authors is a struct with contributor info
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
comment on exported type Author should be of the form "Author ..." (with optional leading article)